src/extension.ts

```diff
@@ -608,6 +611,35 @@ function launchMetals(
         codeLensRefresher
       );
 
+      const istTestManagerDisabled =
+        getValueFromConfig<TestUIKind>(
+          config,
+          "testUserInterface",
+          "Code Lenses"
```

Comment:
Should we have the default value set to Code Lenses?

Reply:
We could default to the `Test Explorer`, no problem.